staging/src/k8s.io/apimachinery/pkg/util/sets/doc.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright The Kubernetes Authors.
+Copyright 2022 The Kubernetes Authors.
```

Comment:
Is it really necessary to add the year here? I would just leave it.

Reply:
Cause I removed `generated` mark, `verify-boilerplate.sh` will think that this file should have the [boilerplate.go.txt](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt) header instead of the [boilerplate.generatego.txt](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.generatego.txt) header.
https://github.com/kubernetes/kubernetes/blob/83c258571686ca4370b2100316c6ef9c99d1ffdd/staging/src/k8s.io/apimachinery/pkg/util/sets/int.go#L16-L18

https://github.com/kubernetes/kubernetes/blob/83c258571686ca4370b2100316c6ef9c99d1ffdd/hack/boilerplate/boilerplate.py#L66-L72